micro-surveys.md

```diff
@@ -0,0 +1,55 @@
+# Micro surveys
+
+The Rust community survey team can run micro-surveys for members of the Rust Project. This document explains how do these micro-surveys work. If you want to run a micro-survey, please create a topic on the survey team [Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey).
+
+## What is a micro survey?
+
+A small survey which is very low cost (in both time and cognitive load) to complete (and prepare). They should be focused on a single topic (possibly with a first question to get context on the respondent for cohort analysis), should take fewer than five minutes to complete and contain just a few (e.g. less than five) questions (ideally all multiple choice rather than free-form, though we might ask a single free-form question if needed). See [below](#examples) for some examples.
+
+The goal is to provide members of the Rust Project a quick way to gauge the opinions of the members of the Rust community about a very targeted topic.
+
+Micro surveys will only be published in English, to reduce the latency of their creation and evaluation.
+
+The community survey team will design and manage the micro-survey. Data will be managed, processed, and distributed in the same way as the annual survey, i.e. raw data will be kept private, some data may be shared publicly in a blog post, processed data will be distributed to relevant teams.
+
+Note that the micro-surveys will be performed by the survey team, without the involvement of the Rust Foundation. That means that the surveys should not ask any privacy or [DEI](https://en.wikipedia.org/wiki/Diversity,_equity,_and_inclusion) questions, to avoid dealing with potential privacy issues.
```

Comment:
@rylev you're probably right, this is more an "internal implementation detail" for us to know. What's important is that we don't need PII or DEI data, so I'd suggest something like:

```suggestion
Note that the micro-surveys should not need to ask any [PII](https://en.wikipedia.org/wiki/Personal_data) or [DEI](https://en.wikipedia.org/wiki/Diversity,_equity,_and_inclusion) questions, to avoid dealing with potential privacy issues and keeping the handling lightweight for us
```

(or something like that)